src/Microsoft.Bcl.HashCode/tests/Microsoft.Bcl.HashCode.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <TargetFramework>$(NetFrameworkMinimum)</TargetFramework>
+    </PropertyGroup>
```

Comment:
You need this here to not run .NET Framework tests on non Windows: https://github.com/dotnet/maintenance-packages/blob/8aa876ae7ba097a9f0eec4dbb9c5d55ce9c02e2d/src/System.Buffers/tests/System.Buffers.Tests.csproj#L6C1-L7C68